lldb/source/Core/Debugger.cpp

```diff
@@ -1875,7 +1883,8 @@ lldb::thread_result_t Debugger::DefaultEventHandler() {
 
   listener_sp->StartListeningForEvents(
       &m_broadcaster, eBroadcastBitProgress | eBroadcastBitWarning |
-                          eBroadcastBitError | eBroadcastSymbolChange);
+                          eBroadcastBitError | eBroadcastSymbolChange |
```

Comment:
See my comment below why we shouldn't listen for `eBroadcastBitProgressCategory` in the default event handler.